sale_report_delivered/reports/sale_report.py

```diff
@@ -165,7 +165,9 @@ def _from(self):
 
     def _where(self):
         return """
-            WHERE sm.state = 'done' OR sm.state IS NULL
+            WHERE (sm.state = 'done' OR sm.state IS NULL) AND
+                  (source_location.usage <> 'customer' AND
```

Comment:
Aren't you discarding with this all returns?

Reply:
Yes, maybe an easier fix could be to set the `signed_qty` to 0 for this special case

Reply:
I think it is better understood this way